docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,105 @@
+  `exchange.hdfs.skip-directory-scheme-validation` property. ({issue}`24627`)
+* Export JMX metric for `blockedQueries`. ({issue}`24907`)
+* {{breaking}} Remove the deprecated precomputed hash optimizer and the related
+  `optimize_hash_generation` configuration property. ({issue}`24792`)
+* Fix failure when using upper-case variable names in SQL user-defined
+  functions. ({issue}`24460`)
+* Prevent failures of the `ARRAY_HISTOGRAM` function resulting from null values.
+  ({issue}`24765`)
+* Prevent wrong JMX metrics values when using dynamic catalogs. ({issue}`24908`)
+
+## JDBC driver
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## CLI
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## Delta Lake connector
+
+* Prevent connection leakage when using the Azure file system. ({issue}`24116`)
+* Deprecate use of the legacy file system support for Azure Storage, Google
+  Cloud Storage, IBM Cloud Object Storage, S3 and S3-compatible object storage
+  systems. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`24878`)
+* Fix incorrect behavior of the `vacuum` procedure deleting unexpected files
+  when the path contains special characters. ({issue}`24872`)
+
+## Faker connector
+
+* Improve similarity of the fake data to the source data when using 
+  [`CREATE TABLE ... AS SELECT`](faker-statistics). ({issue}`24585`)
+
+## Hive connector
+
+* Deprecate use of the legacy file system support for Azure Storage, Google
+  Cloud Storage, IBM Cloud Object Storage, S3 and S3-compatible object storage
+  systems. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`24878`)
+* Prevent connection leakage when using the Azure Storage file system. ({issue}`24116`)
+* Fix NullPointerException when listing tables on Glue. ({issue}`24834`)
+
+## Hudi connector
+
+* Deprecate use of the legacy file system support for Azure Storage, Google
+  Cloud Storage, IBM Cloud Object Storage, S3 and S3-compatible object storage
+  systems. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`24878`)
+* Prevent connection leakage when using the Azure file system. ({issue}`24116`)
+
+## Iceberg connector
+
+* Add the `optimize_manifests ` table procedure. ({issue}`14821`)
+* Allow configuration of the number of commit retries with the
+  `max_commit_retry` table property. ({issue}`22672`)
+* Allow Hive metastore caching. ({issue}`13115`)
```

Comment:
"Add support for caching table metadata" ?

Reply:
I think its only in HMS ..